python/ray/experimental/state/util.py

```diff
@@ -0,0 +1,41 @@
+from typing import Union
+
+
+def convert_string_to_type(
+    val: Union[str, int, float, bool], convert_type: Union[int, float, bool]
```

Comment:
is the type annotation for convert_type correct?

Reply:
I had to add Optional to the `val`. Good catch!